### PR TITLE
LOC CHECKIN | dotnet/roslyn master-vs-deps | 20190110

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Compilers/Core/Portable/xlf/CodeAnalysisResources.de.xlf [hunk 1]
```diff
@@ -678,12 +678,12 @@
...
       <trans-unit id="KeepAliveWithoutShared">
         <source>'/keepalive' option is only valid with '/shared' option.</source>
-        <target state="translated">Die /keepalive-Option ist nur zusammen mit der /shared-Option gültig.</target>
+        <target state="translated">'Die "/keepalive"-Option ist nur zusammen mit der "/shared"-Option gültig.</target>
         <note />
       </trans-unit>
       <trans-unit id="MismatchedVersion">
         <source>Roslyn compiler server reports different protocol version than build task.</source>
-        <target state="translated">Der Roslyn-Compilerserver meldet eine andere Protokollversion als der Buildtask.</target>
+        <target state="translated">Roslyn-Compilerserver meldet unterschiedliche Protokollversion als in Erstellungsaufgabe.</target>
```
Comment (jhinder) [inline src/Compilers/Core/Portable/xlf/CodeAnalysisResources.de.xlf]: "meldet *eine andere* Protokollversion als *die* Erstellungsaufgabe" would be a better translation I think.

File: src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf [hunk 2]
```diff
@@ -84,107 +77,94 @@
       </trans-unit>
       <trans-unit id="String_Escape_Character">
         <source>String - Escape Character</source>
-        <target state="new">String - Escape Character</target>
+        <target state="translated">String - Escape Character</target>
```
Comment (jhinder) [inline src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf]: Not translated?

File: src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf [hunk 3]
```diff
@@ -84,107 +77,94 @@
...
         <note />
       </trans-unit>
       <trans-unit id="Symbol_Static">
         <source>Symbol - Static</source>
-        <target state="new">Symbol - Static</target>
```
Comment (jhinder) [inline src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf]: Where did the translations go?

File: src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf [hunk 4]
```diff
@@ -779,7 +759,7 @@
       </trans-unit>
       <trans-unit id="Rename">
         <source>Rename</source>
-        <target state="translated">Umbenennen</target>
+        <target state="translated">umbenennen</target>
```
Comment (jhinder) [inline src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf]: Why was this changed to lowercase?

File: src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf [hunk 5]
```diff
@@ -844,7 +824,7 @@
       </trans-unit>
       <trans-unit id="Renaming_anonymous_type_members_is_not_yet_supported">
         <source>Renaming anonymous type members is not yet supported.</source>
-        <target state="translated">Das Umbenennen von Membern des Typs anonym wird noch nicht unterstützt.</target>
+        <target state="translated">Das Umbenennen von Elementen des Typs anonym wird noch nicht unterstützt.</target>
```
Comment (jhinder) [inline src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf]: "von Elementen *anonymer Typen*" would be better.

File: src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf [hunk 6]
```diff
@@ -919,7 +899,7 @@
       </trans-unit>
       <trans-unit id="symbol_cannot_be_a_namespace">
         <source>'symbol' cannot be a namespace.</source>
-        <target state="translated">"symbol" kann kein Namespace sein.</target>
+        <target state="translated">'"symbol" kann kein Namespace sein.</target>
```
Comment (jhinder) [inline src/EditorFeatures/Core/xlf/EditorFeaturesResources.de.xlf]: Where did the single quote come from?

File: src/EditorFeatures/CSharp/xlf/CSharpEditorResources.de.xlf [hunk 7]
```diff
@@ -1,10 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="de" original="../CSharpEditorResources.resx">
     <body>
       <trans-unit id="Fix_interpolated_verbatim_string">
         <source>Fix interpolated verbatim string</source>
-        <target state="new">Fix interpolated verbatim string</target>
```
Comment (jhinder) [inline src/EditorFeatures/CSharp/xlf/CSharpEditorResources.de.xlf]: Where did the translation go?

File: src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf [hunk 8]
```diff
@@ -1,75 +1,74 @@
...
       <trans-unit id="Apply_this_qualification_preferences">
         <source>Apply 'this.' qualification preferences</source>
-        <target state="new">Apply 'this.' qualification preferences</target>
+        <target state="translated">Einstellungen zur Qualifikation "this." anwenden</target>
         <note />
       </trans-unit>
       <trans-unit id="Convert_to_method">
         <source>Convert to method</source>
-        <target state="new">Convert to method</target>
+        <target state="translated">Convert to method</target>
```
Comment (jhinder) [inline src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf]: Not translated.

File: src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf [hunk 9]
```diff
@@ -299,7 +296,7 @@
       </trans-unit>
       <trans-unit id="Remove_and_Sort_Usings">
         <source>R&amp;emove and Sort Usings</source>
-        <target state="translated">Using-Direktiven &amp;entfernen und sortieren</target>
+        <target state="translated">R&amp;emove und Art Einsätze</target>
```
Comment (jhinder) [inline src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf]: This translation is gibberish.

File: src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf [hunk 10]
```diff
@@ -574,7 +571,7 @@
       </trans-unit>
       <trans-unit id="_0_does_not_contain_a_constructor_that_takes_that_many_arguments">
         <source>'{0}' does not contain a constructor that takes that many arguments.</source>
-        <target state="translated">"{0}" enthält keinen Konstruktor, der viele Argumente verwendet.</target>
+        <target state="translated">'"{0}" enthält keinen Konstruktor, der viele Argumente verwendet.</target>
```
Comment (jhinder) [inline src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.de.xlf]: "...der *so* viele..." would be better.

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 11]
```diff
@@ -74,37 +69,35 @@
       </trans-unit>
       <trans-unit id="Convert_to_class">
         <source>Convert to class</source>
-        <target state="new">Convert to class</target>
+        <target state="translated">Convert to class</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: Not translated.

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 12]
```diff
@@ -74,37 +69,35 @@
...
       <trans-unit id="Convert_to_linq_call_form">
         <source>Convert to LINQ (call form)</source>
-        <target state="new">Convert to LINQ (call form)</target>
+        <target state="translated">In LINQ konvertieren (Aufrufformular)</target>
         <note />
       </trans-unit>
       <trans-unit id="Convert_to_struct">
         <source>Convert to struct</source>
-        <target state="new">Convert to struct</target>
+        <target state="translated">Convert to struct</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: This and the next one are not translated.

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 13]
```diff
@@ -74,37 +69,35 @@
...
       <trans-unit id="Convert_to_class">
         <source>Convert to class</source>
-        <target state="new">Convert to class</target>
+        <target state="translated">Convert to class</target>
         <note />
       </trans-unit>
       <trans-unit id="Convert_to_linq_call_form">
         <source>Convert to LINQ (call form)</source>
-        <target state="new">Convert to LINQ (call form)</target>
+        <target state="translated">In LINQ konvertieren (Aufrufformular)</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: "Aufrufformular" makes no sense.

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 14]
```diff
@@ -119,97 +112,87 @@
...
       </trans-unit>
       <trans-unit id="Indent_wrapped_arguments">
         <source>Indent wrapped arguments</source>
-        <target state="new">Indent wrapped arguments</target>
         <note />
       </trans-unit>
       <trans-unit id="Indent_wrapped_parameters">
         <source>Indent wrapped parameters</source>
-        <target state="new">Indent wrapped parameters</target>
+        <target state="translated">Einzug für umschlossene Parameter</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: This sounds more like an action than the statement it is now, and "einrücken" would be a better verb here.

File: src/Features/Core/Portable/xlf/FeaturesResources.de.xlf [hunk 15]
```diff
@@ -749,7 +707,7 @@
       </trans-unit>
       <trans-unit id="in_">
         <source>in</source>
-        <target state="translated">in</target>
+        <target state="translated">Ein</target>
```
Comment (jhinder) [inline src/Features/Core/Portable/xlf/FeaturesResources.de.xlf]: I think `in` is meant as a C# keyword here. If so, this must not be translated.

File: src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf [hunk 16]
```diff
@@ -584,12 +575,12 @@
       </trans-unit>
       <trans-unit id="this_preferences_colon">
         <source>'this.' preferences:</source>
-        <target state="translated">this.-Einstellungen:</target>
+        <target state="translated">'this.-Einstellungen:</target>
```
Comment (jhinder) [inline src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf]: Where do the single quotes here and in the next few changes come from?

File: src/VisualStudio/Core/Def/xlf/Commands.vsct.de.xlf [hunk 17]
```diff
@@ -1,10 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="de" original="../Commands.vsct">
     <body>
       <trans-unit id="cmdidContextOrganizeRemoveAndSort|ButtonText">
         <source>Remove &amp;and Sort</source>
-        <target state="translated">Entfernen &amp;und sortieren</target>
+        <target state="translated">Entfernen &und sortieren</target>
```
Comment (jhinder) [inline src/VisualStudio/Core/Def/xlf/Commands.vsct.de.xlf]: The `amp;` is important!

File: src/VisualStudio/Xaml/Impl/xlf/Resources.de.xlf [hunk 18]
```diff
@@ -1,10 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="de" original="../Resources.resx">
     <body>
       <trans-unit id="RemoveAndSortNamespacesWithAccelerator">
         <source>Remove &amp;and Sort Namespaces</source>
-        <target state="translated">N&amp;amespaces entfernen und sortieren</target>
+        <target state="translated">N&amespaces entfernen und sortieren</target>
```
Comment (jhinder) [inline src/VisualStudio/Xaml/Impl/xlf/Resources.de.xlf]: Another `amp;` that went missing.

**Comment by v-zbsail** [comment]
Closing this PR for now as we found a technical issues with XLF/XML validity.
